admin/src/templates/message_test.html

```diff
@@ -0,0 +1,19 @@
+<form action="" method="POST" class="col-xs-8 configuration-form">
+
+  <div class="form-group required" ng-class="{'has-error': errors.message}">
+    <label for="message" translate><translate>Message Body</label>
+    <input id="message" type="text" class="form-control" ng-model="message" />
+    <span class="help-block" ng-if="errors.message" translate>{{errors.message}}</span>
+  </div>
+
+  <div class="form-group required" ng-class="{'has-error': errors.from}">
+    <label for="from" translate><translate>from</label>
+    <input id="from" type="text" class="form-control small" ng-model="from" />
+    <span class="help-block" ng-if="errors.from" translate>{{errors.from}}</span>
```

Comment:
Does this one really get translated?

Reply:
Yeah. First the angular expression is resolved to a translation key, then it's translated.